src/sql/src/func.rs

```diff
@@ -104,6 +104,7 @@ impl TypeCategory {
                 }
             }
             ScalarType::Map { .. } => Self::Pseudo,
+            ScalarType::MzTimestamp => Self::Numeric,
```

Comment:
Should this be `Self::String`? Otherwise it doesn't seem consistent with `from_catalog_type` which the comment indicates it should be.